.vscode/cspell.json

```diff
@@ -210,7 +210,11 @@
       "words": [
         "cref",
         "imds",
-        "IMDS"
+        "IMDS",
+        "Wdac",
+        "Adou",
+        "Drtm",
+        "Hvci"
```

Comment:
what is Hvci and Drtm?

Reply:
these words were causing some build failure due to spelling mistake. Arthur pushed this commit to fix those issues